WeatherService.Tests/BotServiceFactoryTesting/BotServiceFactoryTests.cs

```diff
@@ -7,15 +7,14 @@
 using WeatherService.Utilities;
 
 namespace WeatherService.Tests.BotServiceFactoryTesting;
-public class BotServiceFactoryShould
+public class BotServiceFactoryTests
 {
     private IWeatherBotService _weatherBotService;
 
-    [Theory]
-    [InlineData("RainBot", """{ "enabled": true,"humidityThreshold": 70,"message": "It looks like it's about to pour down!"}""")]
-    public void GetBotService_WhenInputIsRainBotConfiguration_ShouldReturnRainBotService(string botName, string config)
+    [Fact]
```

Comment:
I meant to keep it theory but instead of having 3 methods doing almost the same but for different types, make them one method by using theory and inline data for the 3 bot types